specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1346 @@
+        },
+        "transactionsCount": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Number of transactions for the request."
+        }
+      },
+      "description": "if showStats=true was specified in the request this field will contain information about the request payload."
+    },
+    "DocumentStatistics": {
+      "type": "object",
+      "required": [
+        "charactersCount",
+        "transactionsCount"
+      ],
+      "properties": {
+        "charactersCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of text elements recognized in the document."
+        },
+        "transactionsCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of transactions for the document."
+        }
+      },
+      "description": "if showStats=true was specified in the request this field will contain information about the document payload."
+    },
+    "SentimentConfidenceScorePerLabel": {
+      "type": "object",
+      "required": [
+        "positive",
+        "neutral",
+        "negative"
+      ],
+      "properties": {
+        "positive": {
+          "type": "number",
+          "format": "double"
+        },
+        "neutral": {
+          "type": "number",
+          "format": "double"
+        },
+        "negative": {
+          "type": "number",
+          "format": "double"
+        }
+      },
+      "description": "Represents the confidence scores between 0 and 1 across all sentiment classes: positive, neutral, negative."
+    },
+    "SentenceSentiment": {
+      "type": "object",
+      "required": [
+        "text",
+        "sentiment",
+        "confidenceScores",
+        "offset",
+        "length",
+        "aspects",
+        "opinions"
+      ],
+      "properties": {
+        "text": {
+          "type": "string",
+          "description": "The sentence text."
+        },
+        "sentiment": {
+          "type": "string",
+          "description": "The predicted Sentiment for the sentence.",
+          "enum": [
+            "positive",
+            "neutral",
+            "negative"
+          ],
+          "x-ms-enum": {
+            "name": "SentenceSentimentValue",
+            "modelAsString": false
+          }
+        },
+        "confidenceScores": {
+          "description": "The sentiment confidence score between 0 and 1 for the sentence for all classes.",
+          "$ref": "#/definitions/SentimentConfidenceScorePerLabel"
+        },
+        "offset": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The sentence offset from the start of the document."
+        },
+        "length": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The length of the sentence by Unicode standard."
```

Comment:
We need to be careful here because this may change in the future. Let's remove the Unicode thing. We have a dedicated page for this coming to the docs very soon. We should probably have an aka.ms link to it here and in the offset.

Reply:
Okay, i removed "Unicode" in the Swagger for V3.1-Preview.